app/javascript/app/components/ndcs/ndcs-enhancements-table/ndcs-enhancements-table-selectors.js

```diff
@@ -0,0 +1,111 @@
+import { createSelector } from 'reselect';
+import { deburrUpper } from 'app/utils';
+import uniqBy from 'lodash/uniqBy';
+import sortBy from 'lodash/sortBy';
+import { europeSlug, europeanCountries } from 'app/data/european-countries';
+import isEmpty from 'lodash/isEmpty';
+
+const getSearch = state => state.search || null;
+const getCountries = state => state.countries || null;
+const getIndicatorsData = state => state.indicators || null;
+const getQuery = state => deburrUpper(state.query) || '';
+
+export const getISOCountries = createSelector([getCountries], countries =>
+  countries.map(country => country.iso_code3)
+);
+
+export const getIndicatorsParsed = createSelector(
+  [getIndicatorsData, getISOCountries],
+  (indicators, isos) => {
+    if (!indicators || !indicators.length) return null;
+    return sortBy(
+      uniqBy(
+        indicators.map(i => {
+          return {
+            label: i.name,
+            value: i.slug,
+            categoryIds: i.category_ids,
+            locations: i.locations
+          };
+        }),
+        'value'
+      ),
+      'label'
+    ).filter(ind => ind.categoryIds.indexOf(11) > -1);
+  }
+);
+
+export const tableGetSelectedData = createSelector(
+  [getIndicatorsParsed, getCountries],
+  (indicators, countries) => {
+    if (!indicators || !indicators.length || !indicators[0].locations)
+      return [];
+
+    const refIndicator = indicators[0];
+
+    return Object.keys(refIndicator.locations).map(iso => {
+      if (refIndicator.locations[iso].label_id !== 237) {
```

Comment:
@kevinlustig : what does this `label_id` refer to, what would you expect to have on the id `237`? I'm running the code locally and the page is not showing anything, so I'm wondering if we have a data mismatch, as I've re-imported the data locally several times I am probably far away from id 237.